tests/baselines/reference/classConstructorAccessibility.js

```diff
@@ -67,16 +68,36 @@ var Generic;
     var D = (function () {
         function D(x) {
             this.x = x;
-        } // error
+        }
         return D;
     }());
     var E = (function () {
         function E(x) {
             this.x = x;
-        } // error
+        }
         return E;
     }());
     var c = new C(1);
-    var d = new D(1);
-    var e = new E(1);
+    var d = new D(1); // error
+    var e = new E(1); // error
 })(Generic || (Generic = {}));
+
+
+//// [classConstructorAccessibility.d.ts]
+declare class C {
+    x: number;
+    constructor(x: number);
+}
+declare class D {
+    x: number;
+    constructor(x);

```

Comment:
You'll need to update the declaration emitter to emit `private` / `protected` appropriately